StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
Currently, It is the needs of FirmwareDownload (or ProductionTool) only in order to distinguish the drive ID (happens when drive power cycled). We could support to hold more information in future by storing them in a container instead of a variable.